challenge.md

```diff
@@ -7,37 +7,38 @@ Write your answers inside this file, where it's indicated by the comments.
 1. We just forked a repo on GitHub.com and want to start working on it locally. What command do we use to do that?
 
 <!-- Answer Starts Here -->
-
+git clone <git@...>
 <!-- Answer Ends Here -->
 
 2. OK, we just wrote some code. What command could we use to see a list of all the changes that have been made since the last commit?
 
 <!-- Answer Starts Here -->
-
+git diff
 <!-- Answer Ends Here -->
 
 3. Oops - it looks like there was a change that we forgot to include in our last commit. How can we revise that commit and fix things?
 
 <!-- Answer Starts Here -->
-
+git commit --amend - to change the commit message
```

Comment:
We probably want to do the first one - doing the second would mean that we'd have to recreate all the changes since the last commit.